keyboards/dz60/keymaps/zepol_layout/keymap.c

```diff
@@ -0,0 +1,25 @@
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+	LAYOUT(
+		KC_GESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_NO, KC_BSPC,
+		KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_BSLS,
+		KC_LCTL, KC_A, KC_S, KC_D, KC_F, KC_G, KC_H, KC_J, KC_K, KC_L, KC_SCLN, KC_QUOT, KC_ENT,
+		KC_LSFT, KC_NO, KC_Z, KC_X, KC_C, KC_V, KC_B, KC_N, KC_M, KC_COMM, KC_DOT, KC_SLSH, KC_RSFT, KC_NO,
+		KC_CAPS, KC_LGUI, KC_LALT, KC_SPC, KC_SPC, KC_SPC, KC_RALT, MO(2), KC_NO, MO(1), KC_RCTL),
+
+	LAYOUT(
+		KC_GRV, KC_F1, KC_F2, KC_F3, KC_F4, KC_F5, KC_F6, KC_F7, KC_F8, KC_F9, KC_F10, KC_F11, KC_F12, KC_TRNS, KC_DEL,
+		KC_TRNS, KC_PGUP, KC_UP, KC_PGDN, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_LEFT, KC_DOWN, KC_RIGHT, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_SCROLLLOCK, KC_PAUSE, KC_HOME, KC_END, KC_INSERT, KC_PSCR, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS,
+		KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS, KC_TRNS),
+
+	LAYOUT(
+		KC_TRNS, M(1), M(2), M(3), M(4), M(5), M(6), M(7), M(8), M(9), M(10), M(11), M(12), KC_TRNS, KC_TRNS,
```

Comment:
You only have 3 layers defined here but yet your Fn keys here go all the way until Layer 12. Is this intentional?

Reply:
@mechmerlin `M(#)` are for the old TMK-style macro keys. This keymap doesn't use them (no `action_get_macro` block), so there's no function here.

@loChris, all these `M(#)` keys should probably be swapped out for `KC_TRNS`. That change would make this layer empty, so realistically you could delete the layer.

Also, depending on whether your DZ60 has a hardware reset switch, you may want to include a `RESET` keycode here, which would be a good use of this layer.

Reply:
@mechmerlin @noroadsleft Hey guys! I apologize for the lack of organization with my commit. 

My first board got here yesterday and I just learned how to use qmk. 

In VSCode I can only see the 3 layers that were initially there and which I started personalizing. I'm going to try to get rid of layer 4-12, if I can access them, because I won't really be using those.

@noroadsleft To be clear: You're suggesting I turn layer 3 into a dedicated RESET layer?

Other than the awkwardness of the layers (and title/message), is there anything else I should change?

Thanks!

Reply:
@loChris

> @noroadsleft To be clear: You're suggesting I turn layer 3 into a dedicated RESET layer?

Pretty much. I personally like to keep keycodes that handle the hardware separate from keycodes that are for typing and software. I have a dedicated layer for changing my base layer, adjusting backlighting, resetting to bootloader, and the like. I used to have my RESET key on a normal function layer, but I'd hit it accidentally all the time and have to replug the board to continue typing.

----

Also, you've filed this pull request from your fork's `master` branch. While this isn't a major issue now, later changes may result in merge conflicts if you try to file another PR because the commit history will have diverged between your master and QMK's. It is highly recommended for QMK development - regardless of what is being done or where - to keep your master updated, but **NEVER** commit to it. Instead, do all your changes in a branch (branches are basically free in Git) and issue PRs from your branches when you're developing.

There are instructions on how to keep your master updated here:

[**Best Practices: Your fork's master: Update Often, Commit Never**](https://docs.qmk.fm/#/newbs_best_practices?id=your-fork39s-master-update-often-commit-never)

If you need any help with this just ask.